datafusion/common/src/config.rs

```diff
@@ -2626,7 +2665,7 @@ impl Display for OutputFormat {
 mod tests {
     use crate::config::{
         ConfigEntry, ConfigExtension, ConfigField, ConfigFileType, ExtensionOptions,
-        Extensions, TableOptions,
+        Extensions, TableOptions, TableParquetOptions,
```

Comment:
This line is what has caused the build failure. Because you've imported the `TableParquetOptions` name, places where it is already used in the `tests` module in fully qualified form (`crate::config::TableParquetOptions`) now raise an error.

You can reproduce it if you run the same command as CI:
```
cargo check --profile ci --workspace --all-targets --features integration-tests --locked
```

To fix it, you can either revert this change and refer to it as `crate::config::TableParquetOptions` in the new test, or make the change suggested in the error.